README.md

```diff
@@ -154,15 +193,15 @@ Write Ruby code that will create an instance of a person.
 Your answer...
 
 ```ruby
-Replace this with your answer
+trump = Person.new("Donald", 70)
```

Comment:
You need to indicate with attributes gets values of "Donald" and 70. Some options...

```rb
# option A
trump = Person.new(name: "Donald", age: 70)
trump.save # .new doesn't automatically save it

# option B
Person.create(name: "Donald", age: 70) # .create = .new + .save
```